DrawerView/DrawerView.swift

```diff
@@ -174,6 +174,20 @@ private struct ChildScrollViewInfo {
             updateVisuals()
         }
     }
+    
+    /// The background blur effect style for dark mode.
+    public var darkBlurEffectStyle: UIBlurEffect.Style = .dark {
+        didSet {
+            updateVisuals()
+        }
+    }
+    
+    /// The background blur effect style for light mode.
+    public var lightBlurEffectStyle: UIBlurEffect.Style = .light {
```

Comment:
Sure, for some reason developers may set a dark style for light mode, but, seems like we shouldn't make a limitation on it, because for some kind of Themes, developers may want to set `.light` for dark mode and `.extraLight` for light mode. You know what I mean? :)